Felienne/hedy
# [UI IMPROVEMENT] Centralizes workflow for adding students to classes

**Description**
In this PR we continue with the clean-up of the class overview page to make it less crowded with a more logical workflow. The multiple accounts creation section is removed from the for-teachers page and instead placed within the class overview page. We create a dedicated "add students" section, which displays multiple options when clicked.

**Fixes**
This PR fixes #2869.

**How to test**
Navigate to the class overview page, verify that all three options are working as expected.

**New view**
![Schermafbeelding 2022-06-10 om 12 23 03](https://user-images.githubusercontent.com/38155424/173046042-7f67b5f0-22e6-40aa-aae6-27c342dea2e6.png)
![Schermafbeelding 2022-06-10 om 12 23 10](https://user-images.githubusercontent.com/38155424/173046046-330cf75d-beba-4c89-911d-6ddd7d28dee0.png)
